### ISSUE TITLE
Allow custom backup location

### ISSUE THREAD

**Issue by PandaCoderPL** [zoff99/ToxAndroidRefImpl#307]
**Is your feature request related to a problem? Please describe.**
No

**Describe the solution you'd like**
I would like to be able to specify custom backup location so I can export Tox savedata to selected folder.

**Your system information:**
 - TRIfA Version: 1.0.123
 - App from Google-Play / F-Droid / Github-Releases / other: F-Droid
 - Android OS Version: Android 12 (GrapheneOS)
 - Device: Google Pixel 4a

**Comment by gogo2464** [comment]
assign me to this issue please

**Comment by zoff99** [comment]
@gogo2464 why? do you want to work on it? you can just open a PR in this repo. no need to assign anyone

**Comment by gogo2464** [comment]
good point. I ask no assignment anymore. I go PR. It will proove my serious.
